Manual.md

```diff
@@ -781,6 +781,12 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gem` For packages that are installed using gemspec files or gems
+from [RubyGems](https://rubygems.org/). It is expected that packages
```

Comment:
`Packages that use this build style must follow the ruby-<pkgname> naming convention, with....`

Reply:
fixed, thanks